### PR TITLE
Create resource sharing

### PR DESCRIPTION
**Pull request by Frosina-lab**

### CODE REVIEW

File: database/migrations/2014_10_12_200000_create_resources_table.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use Illuminate\Database\Migrations\Migration;
+
+class CreateResourcesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()	//ресурси за табела
```
Comment (TsTrv) [inline database/migrations/2014_10_12_200000_create_resources_table.php:14]:
```suggestion
    public function up()
```

File: database/migrations/2014_10_12_200000_create_resources_table.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        Schema::create('resources', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('content');
+            $table->string('url');
+            $table->string('tags');
+            $table->unsignedInteger('user_id');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
+
```
Comment (TsTrv) [inline database/migrations/2014_10_12_200000_create_resources_table.php:24]:
```suggestion
```

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Најава') }}</div>
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:8]:
```suggestion
                <div class="card-header">{{ __('Login') }}</div>
```

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('login') }}">
+                        @csrf
+
+                        <div class="form-group row">
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Е-маил Адреса') }}</label>
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:15]:
```suggestion
                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Email Address') }}</label>
```

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                                @if ($errors->has('email'))
+                                    <span class="invalid-feedback" role="alert">
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group row">
+                            <label for="password" class="col-md-4 col-form-label text-md-right">{{ __('Лозинка') }}</label>
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:29]:
```suggestion
                            <label for="password" class="col-md-4 col-form-label text-md-right">{{ __('Password') }}</label>
```

File: resources/views/auth/login.blade.php [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                            </div>
+                        </div>
+
+                        <div class="form-group row">
+                            <div class="col-md-6 offset-md-4">
+                                <div class="form-check">
+                                    <input class="form-check-input" type="checkbox" name="remember" id="remember" {{ old('remember') ? 'checked' : '' }}>
+
+                                    <label class="form-check-label" for="remember">
+                                        {{ __('Запомни') }}
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:48]:
```suggestion
                                        {{ __('Remember me') }}
```

File: resources/views/auth/login.blade.php [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+                                    </label>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="form-group row mb-0">
+                            <div class="col-md-8 offset-md-4">
+                                <button type="submit" class="btn btn-primary">
+                                    {{ __('Најави се') }}
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:57]:
```suggestion
                                    {{ __('Login') }}
```

File: resources/views/auth/login.blade.php [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+                                </button>
+
+                                @if (Route::has('password.request'))
+                                    <a class="btn btn-link" href="{{ route('password.request') }}">
+                                        {{ __('Ја заборави лозинката?') }}
```
Comment (TsTrv) [inline resources/views/auth/login.blade.php:62]:
```suggestion
                                        {{ __('Forgot password?') }}
```

File: resources/views/auth/passwords/email.blade.php [hunk 9]
```diff
@@ -0,0 +1,47 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Обнови Лозинка') }}</div>
```
Comment (TsTrv) [inline resources/views/auth/passwords/email.blade.php:8]:
```suggestion
                <div class="card-header">{{ __('Reset password') }}</div>
```

File: resources/views/auth/passwords/email.blade.php [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+                        <div class="alert alert-success" role="alert">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    <form method="POST" action="{{ route('password.email') }}">
+                        @csrf
+
+                        <div class="form-group row">
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Е-Маил Адреса') }}</label>
```
Comment (TsTrv) [inline resources/views/auth/passwords/email.blade.php:21]:
```suggestion
                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Email address') }}</label>
```

File: resources/views/auth/passwords/email.blade.php [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group row mb-0">
+                            <div class="col-md-6 offset-md-4">
+                                <button type="submit" class="btn btn-primary">
+                                    {{ __('Испратими линк за обнова на лозинка') }}
```
Comment (TsTrv) [inline resources/views/auth/passwords/email.blade.php:37]:
```suggestion
                                    {{ __('Get a new passwrod reset link') }}
```

File: resources/views/auth/passwords/reset.blade.php [hunk 12]
```diff
@@ -0,0 +1,65 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Обнови Лозинка') }}</div>
```
Comment (TsTrv) [inline resources/views/auth/passwords/reset.blade.php:8]:
```suggestion
                <div class="card-header">{{ __('Reset password') }}</div>
```

File: resources/views/auth/passwords/reset.blade.php [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('password.update') }}">
+                        @csrf
+
+                        <input type="hidden" name="token" value="{{ $token }}">
+
+                        <div class="form-group row">
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Е-Маил Адреса') }}</label>
```
Comment (TsTrv) [inline resources/views/auth/passwords/reset.blade.php:17]:
```suggestion
                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('Email address') }}</label>
```

File: resources/views/auth/passwords/reset.blade.php [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+                                @if ($errors->has('password'))
+                                    <span class="invalid-feedback" role="alert">
+                                        <strong>{{ $errors->first('password') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group row">
+                            <label for="password-confirm" class="col-md-4 col-form-label text-md-right">{{ __('Потврди Лозинка') }}</label>
```
Comment (TsTrv) [inline resources/views/auth/passwords/reset.blade.php:45]:
```suggestion
                            <label for="password-confirm" class="col-md-4 col-form-label text-md-right">{{ __('Confirm password') }}</label>
```

File: resources/views/auth/passwords/reset.blade.php [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+                            <div class="col-md-6">
+                                <input id="password-confirm" type="password" class="form-control" name="password_confirmation" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group row mb-0">
+                            <div class="col-md-6 offset-md-4">
+                                <button type="submit" class="btn btn-primary">
+                                    {{ __('Обнови Лозинка') }}
```
Comment (TsTrv) [inline resources/views/auth/passwords/reset.blade.php:55]:
```suggestion
                                    {{ __('Reset password') }}
```

File: resources/views/welcome.blade.php [hunk 16]
```diff
@@ -81,7 +81,7 @@
 
             <div class="content">
                 <div class="title m-b-md">
-                    Laravel
+                   Welcome Bitches :)
```
Comment (TsTrv) [inline resources/views/welcome.blade.php:84]: 😱

**Review by TsTrv — APPROVED** [review]

**Review by Frosina-lab — COMMENTED** [review]
Changes are made

**Pull request merged**